From 6e9f130a3be14a92a16b2332f6c92f873ebb8af5 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Tue, 3 Sep 2019 15:57:49 +0800
Subject: [PATCH] net: tsn: fix headfile voliates the new rule

After commit
622445541b75 ("kbuild: detect missing "WITH Linux-syscall-note" for uapi headers")
the headfile must explicitly include "WITH Linux-syscall-note".

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 include/uapi/linux/tsn.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/uapi/linux/tsn.h
+++ b/include/uapi/linux/tsn.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause) */
+/* SPDX-License-Identifier: (GPL-2.0+ OR BSD-3-Clause) WITH Linux-syscall-note */
 /* Copyright 2017-2019 NXP */
 
 #ifndef __UAPI_GENL_TSN_H
